addons/hr_timesheet/models/project_project.py

```diff
@@ -138,26 +142,43 @@ def _compute_total_timesheet_time(self):
             total_time *= project.timesheet_encode_uom_id.factor
             project.total_timesheet_time = int(round(total_time))
 
+    def copy(self, default=None):
+        new_projects = super().copy(default=default)
+        analytic_fnames = self._get_plan_fnames()[1:]
+        for old_project, new_project in zip(self, new_projects):
+            if new_project.allow_timesheets:
+                new_project.write({
+                    fname: old_project[fname].id
+                    for fname in analytic_fnames
+                })
```

Comment:
It should be copied by default, why are we doing it manually?
https://github.com/odoo/odoo/blob/9b15521e19335113e73d74a9b1188739c74cb736/addons/analytic/models/analytic_plan.py#L278

Reply:
Indeed, it's a mistake